packages/ui-emulator/angular.json

```diff
@@ -0,0 +1,145 @@
+                }
+            },
+            "architect": {
+                "build": {
+                    "builder": "@angular-devkit/build-angular:browser",
+                    "options": {
+                        "outputPath": "dist/cloud-director-container",
+                        "index": "src/index.html",
+                        "main": "src/main.ts",
+                        "polyfills": "src/polyfills.ts",
+                        "tsConfig": "src/tsconfig.app.json",
+                        "assets": [
+                            "src/favicon.ico",
+                            "src/assets"
+                        ],
+                        "styles": [
+                            "node_modules/@clr/icons/clr-icons.min.css",
+                            "node_modules/@clr/ui/clr-ui.min.css",
+                            "src/styles.scss"
+                        ],
+                        "scripts": [
+                            "node_modules/@webcomponents/custom-elements/custom-elements.min.js",
+                            "node_modules/@clr/icons/clr-icons.min.js",
+                            "node_modules/systemjs/dist/system.src.js"
+                        ]
+                    },
+                    "configurations": {
+                        "production": {
+                            "fileReplacements": [
+                                {
+                                    "replace": "src/environments/environment.ts",
+                                    "with": "src/environments/environment.prod.ts"
+                                }
+                            ],
+                            "optimization": true,
+                            "outputHashing": "all",
+                            "sourceMap": false,
+                            "extractCss": true,
+                            "namedChunks": false,
+                            "aot": true,
+                            "extractLicenses": true,
+                            "vendorChunk": false,
+                            "buildOptimizer": true
+                        }
+                    }
+                },
+                "serve": {
+                    "builder": "@angular-devkit/build-angular:dev-server",
+                    "options": {
+                        "browserTarget": "cloud-director-container:build",
+                        "proxyConfig": "src/proxy.conf.json"
+                    },
+                    "configurations": {
+                        "production": {
+                            "browserTarget": "cloud-director-container:build:production"
+                        }
+                    }
+                },
+                "extract-i18n": {
+                    "builder": "@angular-devkit/build-angular:extract-i18n",
+                    "options": {
+                        "browserTarget": "cloud-director-container:build"
+                    }
+                },
```

Comment:
We may not need the i18n extraction since the plugins has their own, do we plan to leverage the angular one?

Reply:
Not sure about this as well, since its coming from the developer container.